### DISCUSSION TITLE
How to get "fmw_version_coherence.jar" to install Oracle Coherence Community Edition on Linux Server?

### DISCUSSION THREAD

**Discussion by takeaki-m** [oracle/coherence#118] [Q&A] [answered]
## Goal
I want to install Oracle Coherence Community Edition to Linux Server.

Due to terms of my work, I can't use container images, can use only installing on server.

## What I did

I'm proceeding with the installation following the instructions in the link below.
https://docs.oracle.com/en/middleware/standalone/coherence/14.1.1.0/install/installing-oracle-coherence-java.html#GUID-2902C0B4-9684-4328-AA17-249882242819

I used `fmw_14.1.1.0.0_coherence.jar` got from the link below.
https://www.oracle.com/middleware/technologies/coherence-downloads.html

## Problem
I succeeded to installing and tried to start up coherence servers.
However, at logs, I found that the coherence I installed is GE, not a CE(Community Edition).

e.g.
```
2024-03-29 16:00:18.951/3339.929 Oracle Coherence GE 14.1.1.0.0 <Info>
```

## Want to do
I want to use Oracle Coherence Community Edition, not GE.
So Where can I get Community Edition installer firmware "fmw_14.1.1.0.0_coherence.jar" ?

or installer is correct, but my operation may have a problem?

(Is the link below different from the link above?)
Oracle Fusion Middleware Software Downloads
https://www.oracle.com/middleware/technologies/fusionmiddleware-downloads.html

**Comment by thegridman** [comment, marked as answer]
@takeaki-m The instruction you linked to are for the commercial version of Coherence which Oracle delivers via an installer. Coherence CE is not delivered that way, there is no Oracle Installer. 

Coherence CE is open source and distributed on Maven Central. You use Coherence CE just like most open source applications and add it as a dependency to your project. See the docs https://coherence.community/latest/24.03/docs/#/docs/about/03_quickstart

In a lot of cases having an installer for Coherence is pointless as really Coherence is a set of libraries that your application depends on. In all my many years of building Coherence applications I have never run Coherence from the installer location, it has always been built into my application distribution.

**Reply by takeaki-m** [reply]
@thegridman Thanks a comment. I didn't tell commercial document and community one,  but your explanation helped me understand.
So I'll run application with coherence dependency as ordinal Java applications!.
